src/main/java/uk/gov/hmcts/sptribs/ciccase/tab/CaseTypeTab.java

```diff
@@ -59,6 +63,21 @@ private void buildCaseFlagTab(ConfigBuilder<CaseData, State, UserRole> configBui
         }
     }
 
+    private void buildCaseLinkTab(ConfigBuilder<CaseData, State, UserRole> configBuilder) {
+        if (caseLinkEnabled) {
+            doBuildCaseLinkTab(configBuilder);
+        }
+    }
+
+    private void doBuildCaseLinkTab(ConfigBuilder<CaseData, State, UserRole> configBuilder) {
+        configBuilder.tab("caseLinks", "Linked cases")
+            .forRoles(ST_CIC_CASEWORKER, ST_CIC_SENIOR_CASEWORKER, ST_CIC_HEARING_CENTRE_ADMIN,
```

Comment:
We followed the exact steps provided in confluence page of Case Linking feature as we discussed with CCD team. Not sure it would if we change the showCondition to tab. So we would like to keep as is for now.